<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditComandForm</class>
 <widget class="QWidget" name="EditComandForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Изменить команду</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Система команд</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="systemsComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Команда</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comandsComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Значение</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="comandValue"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="editComandButton">
     <property name="text">
      <string>Изменить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
